rpcd/etc/openstack_deploy/user_variables.yml

```diff
@@ -82,3 +82,8 @@ nova_ceilometer_enabled: False
 swift_ceilometer_enabled: False
 keystone_ceilometer_enabled: False
 tempest_service_available_ceilometer: False
+
+neutron_git_repo: "https://git.openstack.org/openstack/neutron"
+neutron_git_install_branch: "b80fa23ac0b292db84df84855141eb16a4e0fec0"
+neutron_git_dest: "/opt/neutron_{{ neutron_git_install_branch | replace('/', '_') }}"
```

Comment:
@prometheanfire That's what the initial patchset was, but it's since changed, so I'm wondering if that assumption is false (fwiw, I think that's what should be needed).

Actually, looking at the commit message:

> neutron_git_repo is required so that the lookup filter py_pkgs knows to
look in user_variables.yml for the new SHA.

So it's to account for behavior in OSA.